maestrowf/datastructures/core/study.py

```diff
@@ -193,8 +255,8 @@ class Study(DAG, PickleInterface):
       using the same environment.
     """
 
-    def __init__(self, name, description,
-                 studyenv=None, parameters=None, steps=None, out_path="./"):
+    def __init__(self, name, description, studyenv=None,
+                 parameters=None, steps=None, out_path="./"):
```

Comment:
Since you're passing the study environment here, the output path might be redundant. Could be an opportunity to clean up some parameters here.